src/Model/Address.php

```diff
@@ -92,7 +92,9 @@ public function validate()
      */
     public function getStreetAddress(): ?string
     {
-
+        if ($this->streetAddress > 50 ) {
```

Comment:
Should this rather be
`if (mb_strlen($this->streetAddress) > 50)` ? Using `mb_Strlen()` is adviced, since address can contain ö/ä, which can fail `strlen()` comparison, even there's under 50 character.

Also exception should specify limitation, so user knows message why validation fails.

Reply:
To make sure it works also when internal encoding is not set to UTF-8, that should be `if (mb_strlen($this->streetAddress, 'UTF-8') > 50)`.